test/integration/framework/master_utils.go

```diff
@@ -342,12 +342,12 @@ func NewControlPlaneConfigWithOptions(opts *MasterConfigOptions) *controlplane.C
 type CloseFunc func()
```

Comment:
Should MasterConfigOptions be renamed to controlPlaneConfigOptions as well?

Reply:
Of course it could, but since this variable involves many files, I just changed it for review purposes.

ref https://github.com/kubernetes/kubernetes/pull/100181#issuecomment-805017091 commented.

Reply:
maybe a follow-up commit?
I think it's best if we can clean up all variables in a subset of the project instead of one to a PR 😅

Reply:
I'm sorry for the late reply. I should have made the modification in a follow-up commit, but I didn't spare time to make the modification in these two days and  the PR has been merged, so I will make the modification in the following PR.
Thanks.😁